.github/workflows/win-exe.yml

```
@@ -284,6 +301,8 @@ jobs:
     name: Build .exe installer
     runs-on: ${{ matrix.os }}
     needs: [ get-version, build-msi ]
+    outputs:
+      sha256sum: ${{ steps.sha256sum.outputs.value }}
```

actionlint (1.7.11)

[error] property "sha256sum" is not defined in object type {}

(expression)